generators/server/templates/_gradle.properties

```diff
@@ -13,6 +13,8 @@ cassandra_unit_spring_version=3.0.0.1
 <%_ if (cucumberTests) { _%>
 cucumber_version=1.2.4
 <%_ } _%>
+# Overridden to get metrics-jcache
+dropwizard-metrics.version=3.2.2
```

Comment:
I tried. Doesn't work. However, I'm using the dependencyManagement version of metrics-core instead of the variable. Also, I've decided to always add metrics-jcache. That was on purpose. But I can indeed add it only when needed. Let me change it.